packages/cicero-core/src/templateloader.js

```diff
@@ -67,6 +67,7 @@ class TemplateLoader extends FileLoader {
         const requestContents = await TemplateLoader.loadZipFileContents(zip, 'request.json', true);
         const packageJsonObject = await TemplateLoader.loadZipFileContents(zip, 'package.json', true, true);
         const grammar = await TemplateLoader.loadZipFileContents(zip, 'text/grammar.tem.md', false, false);
+        const authorSignature = await TemplateLoader.loadZipFileContents(zip, 'signatures.json', true, false);
```

Comment:
What happens for non-signed archives?

Reply:
Even the non-signed archives get loaded. The `signatures.json` is not compulsory. the `required` param is `false`.

Reply:
Ah I forgot the required param. Got it, thanks!